weavetest/assert/assert.go

```diff
@@ -0,0 +1,40 @@
+package assert
+
+import (
+	"reflect"
+	"testing"
+)
+
+// Nil fails the test if given value is not nil.
+func Nil(t testing.TB, value interface{}) {
+	t.Helper()
+	if !isNil(value) {
+		t.Fatalf("want a nil value, got %#v", value)
```

Comment:
Argh.

I *really* like `%+v` here.

Am I the only one who ever uses those stack traces in test failures to debug?

Fine with generalizing, but 90% of the usages were for errors and keep the flow as is, but I much prefer `s/%#v/%+v/` (which is the same as `%v` for all non "special" errors)

Reply:
> Am I the only one who ever uses those stack traces in test failures to debug?

Yes :joy: 

I will make it `%+v`.